features/tags_and_wrangling/tag_wrangling.feature

```diff
@@ -335,7 +335,9 @@ Feature: Tag wrangling
       And I should see "Grandparent" within "#parent_MetaTag_associations_to_remove_checkboxes"
       But I should not see "Parent" within "#parent_MetaTag_associations_to_remove_checkboxes"
 
-    When I view the tag "Child"
+    # Burst template cache
+    When it is currently 25 hours from now
+      And I view the tag "Child"
```

Comment:
This works because it expires [the cache](https://github.com/otwcode/otwarchive/blob/f2de69b0f3de9b3ab3c01ece61b85ebc2c0c961c/app/views/tags/show.html.erb#L73) based on time. Another way to expire the cache is to make the [cache_key](https://api.rubyonrails.org/classes/ActiveRecord/Integration.html#method-i-cache_key) different. The archive uses the cache keys versioned by updated_at, so if at least one second passes between the creation and the edit of the tag, the updated_at will be different and with it the cache key.

So, waiting [for one second](https://github.com/Bilka2/otwarchive/commit/fccf6fe08a113028cfac3aa407ac2be53fa0c706) before the "Save changes" step also [does the job](https://github.com/Bilka2/otwarchive/actions/runs/6836015833). I don't know what's preferred here, but this method would be consistent with the other flaky test fixes.

Reply:
Yeah, let's use one second to be consistent with fixes elsewhere.